base/stats.go

```diff
@@ -16,13 +52,39 @@ const (
 //to find valid property names
 //
 func init() {
-	StatsExpvars.Add("changesFeeds_total", 0)
-	StatsExpvars.Add("changesFeeds_active", 0)
-	StatsExpvars.Add("requests_total", 0)
-	StatsExpvars.Add("requests_active", 0)
-	StatsExpvars.Add("revisionCache_hits", 0)
-	StatsExpvars.Add("revisionCache_misses", 0)
+	for _, stat := range globalStatNames {
+		StatsExpvars.Add(stat, 0)
+	}
 	TimingExpvars = NewSequenceTimingExpvar(KTimingExpvarFrequency, KTimingExpvarVbNo, "st")
 	StatsExpvars.Set("sequenceTiming", TimingExpvars)
+	InitDbStats("_server")
+
+	ticker := time.NewTicker(statLogInterval)
+	go func () {
+		for {
+			<-ticker.C
+			Statsf(KeyAll, StatsExpvars.String())
```

Comment:
Using StatsExpvars to define the set of logged expvars is a decent way of managing the set of expvars we're sending to logs, but makes it more challenging to incorporate elements of built-in expvars (e.g. memstats) that we may want (now or in future).